### ISSUE TITLE
Add Webservice Request: ZIMBRA

### ISSUE THREAD

**Issue by dehylton** [Noitidart/MailtoWebmails#22]
I was trying to install a webmail service from the options panel of MailtoWebesrvices but the one I was looking for was not found. I was looking for ZIMBRA.

**Comment by Noitidart** [comment]
Please install the addon from this branch it supports Zimbra:

https://github.com/Noitidart/MailtoWebmails/issues/15

To install directly from github install this addon https://addons.mozilla.org/en-US/firefox/addon/github-extension-installer/

Then install straight from the repo: https://github.com/Noitidart/MailtoWebmails/tree/%2315-Zimbra

**Comment by dehylton** [comment]
would it be possible to provide a way to enter our own service/url combinations? for example, in my case the zimbra server is my own company's so the generic zimbra setting won't work. it didn't occur to me this would be an issue, but it's fairly obvious now and getting around it will require coding on your part. a plugin that does this is available for chrome already (works on my chromebook) so the code is already out there somewhere. 

for now, i've simply edited the xpi file and added my own stuff in place of the fastmail config. works like a charm, but of course is limited. 

thanks for your work. 


----- On May 9, 2015, at 4:29 PM, Noitidart <notifications@github.com> wrote: 



Please install the addon from this branch it supports Zimbra: 

#15 


To install directly from github install this addon https://addons.mozilla.org/en-US/firefox/addon/github-extension-installer/ 


Then install straight from the repo: https://github.com/Noitidart/MailtoWebmails/tree/%2315-Zimbra

**Comment by Noitidart** [comment]
Yes absolutley that's a great idea and it's something I plan on for a future release I haven't go to work on this addon too much but I want to soon as lots of people are requesting lots of different ones.

**Comment by Noitidart** [comment]
This is the issue tracker for the feature you mentioned, i hope to land in next version: https://github.com/Noitidart/MailtoWebmails/issues/11

**Comment by Noitidart** [comment]
Hey @dehylton  v2 is coming out this week or next week, please prepare your urls you can submit and share with the community in this version :)
